server/services/courtRegisterService.ts

```diff
@@ -138,17 +138,63 @@ export default class CourtRegisterService {
     context: Context,
     courtId: string,
     buildingId: string,
-    contacts: UpdateCourtBuildingContact | { id?: string }[]
+    contacts: NewOrExistingContact[]
   ): Promise<void> {
     logger.info(`Amending contacts for court ${courtId} building for ${buildingId}`)
     const token = await this.hmppsAuthClient.getApiClientToken(context.username)
     const building = (await CourtRegisterService.restClient(token).get({
       path: `/courts/id/${courtId}/buildings/id/${buildingId}`,
     })) as CourtBuilding
 
-    // TODO update each contact, add new ones and delete others
-    logger.info(`existing contacts ${building.contacts}`)
-    logger.info(`contacts to be updated ${contacts}`)
+    const same = (updatedContact: NewOrExistingContact, originalContact: CourtBuildingContact) =>
+      originalContact.id.toString(10) === updatedContact.id
```

Comment:
What does `toString(10)` do?

Reply:
radix, base 10

Reply:
maybe I should convert the number to a string for clarity

Reply:
It confused me a bit because base 10 is the default so I started to wonder if that parameter was doing something else.  I think `id.toString()` would be clearer.